report_alternative_layout/report/report_template.xml

```diff
@@ -0,0 +1,158 @@
+                    t-value="report.with_context(lang=report_partner.lang)"
+                />
+                <div class="col-9 text-end">
+                    <!-- Display the header only if 'report_header' is set.
+                    Handle auto-inserted <p><br></p> from HTML fields -->
+                    <t
+                        t-if="company.report_header and company.report_header != '&lt;p&gt;&lt;br&gt;&lt;/p&gt;'"
+                    >
+                        <span t-field="company.report_header" name="moto" />
+                    </t>
+                    <!-- Display the document number in header -->
+                    <t t-if="report and report.show_document_number">
+                        <div class="col text-end" t-if="o.name">
+                            <strong>
+                                Document No:
+                                <span t-field="o.name" />
+                            </strong>
+                        </div>
+                    </t>
+                    <!-- Display the date field in header -->
+                    <t t-if="report and report.date_field_id and o">
+                        <t t-set="date_value" t-value="report._get_date_value(o)" />
+                        <t t-if="date_value">
+                            <div class="col text-end">
+                                <span t-esc="report._get_date_field_label()" />:<span
+                                >&#xa0;</span>
+                                <span t-esc="date_value" />
+                            </div>
+                        </t>
+                    </t>
+                </div>
+            </t>
+        </xpath>
+        <!-- Hide company address in the header if alternative layout is selected -->
+        <xpath expr="//div[@name='company_address']/ul" position="attributes">
+            <attribute name="t-if">not apply_alternative_layout</attribute>
+        </xpath>
+        <!-- Display address in the header when 'show_address_in_header' is enabled -->
+        <xpath
+            expr="//div[hasclass('row')][div[@name='company_address']]"
+            position="after"
+        >
+            <t t-if="show_address_in_header">
+                <div class="row">
+                    <div class="col-12">
+                        <t
+                            t-call="report_alternative_layout.report_custom_address_layout"
+                        />
+                    </div>
+                </div>
+            </t>
+        </xpath>
+        <!-- By disabling the default margins,
+        the starting position of the first page and subsequent pages will be aligned,
+        making it easier to adjust layout using the paper format. -->
+        <xpath expr="//div[hasclass('pt-5')]" position="attributes">
+            <attribute name="class" remove="pt-5" />
+            <attribute
+                name="t-att-class"
+            >'pt-5' if not apply_alternative_layout else ""</attribute>
+        </xpath>
```

Comment:
Can you please check if this works? Relying on the `pt-5` class makes the design a bit fragile.
```suggestion
        <xpath expr="//t[@t-call='web.address_layout']/.." position="attributes">
            <attribute name="t-att-class">'pt-5' if not apply_alternative_layout else ''</attribute>
        </xpath>
```

Reply:
It works.

Reply:
@AungKoKoLin1997 Can you please do the same to the 18.0 module?